onnx/defs/schema.h

```diff
@@ -21,6 +21,8 @@
 
 namespace ONNX_NAMESPACE {
 
+struct Node;
```

Comment:
one question with using Node - does it support standalone creation (without owning Graph?)

I also wonder whether Node can be refactored a bit to have only minimal set of helper functions for accessing attributed, not all graph-associated machinery in this case. The concern is of keeping the schemas part very light-weight. But maybe it's ok